google-cloud-pubsub/lib/google/cloud/pubsub/topic.rb

```diff
@@ -486,41 +486,23 @@ def delete
         #   retry_policy = Google::Cloud::PubSub::RetryPolicy.new minimum_backoff: 5, maximum_backoff: 300
         #   sub = topic.subscribe "my-topic-sub", retry_policy: retry_policy
         #
-        def subscribe subscription_name,
-                      deadline: nil,
-                      retain_acked: false,
-                      retention: nil,
-                      endpoint: nil,
-                      push_config: nil,
-                      labels: nil,
-                      message_ordering: nil,
-                      filter: nil,
-                      dead_letter_topic: nil,
-                      dead_letter_max_delivery_attempts: nil,
-                      retry_policy: nil
+        def subscribe subscription_name, options = {}
```

Comment:
This is a  breaking usage change. Options hash is different from keyword arguments in Ruby 3.

Reply:
Have updated this to use `**options` which represents the keyword arguments.